### ISSUE TITLE
IOU details - "&nbsp;" html tag is visible near currency abbreviation in IOU details

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15576]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open ND App
2. Create new money request with currency without special symbol, but with abbreviation only: UAH, AFN, AMD etc
3. check IOU details

## Expected Result:
The user should be able to check IOU details without any issues

## Actual Result:
Html tag “`&nbsp`;” is visible near currency abbreviation in request money IOU details

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** 1.2.77-3
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/222002604-0f419845-f4d6-4a2d-9927-f55f9fe41b7b.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @abekkala (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Triggered auto assignment to @yuwenmemon (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by yuwenmemon** [comment]
Not able to reproduce on Dev. Gonna see what's different between it and staging.

**Comment by yuwenmemon** [comment]
@pecanoro I think this is related to https://github.com/Expensify/App/issues/15131

**Comment by yuwenmemon** [comment]
Looks like @pecanoro is probably out for the day. I think I have a fix for this. @aldo-expensify since you reviewed the PR that's a part of the issue I linked would you mind reviewing this one?

**Comment by mvtglobally** [comment]
@yuwenmemon check this one as well https://github.com/Expensify/Web-Expensify/pull/36598. new PR added as well

**Comment by yuwenmemon** [comment]
@mvtglobally Yeah that's the one I'm linking (Issue instead of PR)

**Comment by pecanoro** [comment]
Wait, this was on purpose, we needed this in order to fix this: https://github.com/Expensify/App/pull/15340

**Comment by pecanoro** [comment]
Left a comment in the PR, I should have warned Applause, my fault, sorry about it

**Comment by aldo-expensify** [comment]
@pecanoro My thoughts were:

- @yuwenmemon 's PR was fixing the issue with the `&nbsp;` after the currency, and your back end PR is still fixing that the first name in the IOU preview shows as html entity codes
- The contributor's front end PR https://github.com/Expensify/App/pull/15340/files doesn't seem to be fixing the case @yuwenmemon 's PR fixed

**Comment by yuwenmemon** [comment]
@pecanoro so what should we be doing? Reverting?

**Comment by yuwenmemon** [comment]
Nvm, looks like we're effectively "reverting" in this PR: https://github.com/Expensify/App/pull/15340

**Comment by pecanoro** [comment]
I think your PR fixed this deploy blocker and the problems with the message, he still needs to fix the profile names, so I think it's fine. Your PR was CPed, right?

**Comment by yuwenmemon** [comment]
Correct! Okay sounds good. We'll just move along 👌

**Comment by yuwenmemon** [comment]
This has been CP'ed to staging

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.77-4 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15578


If no regressions arise, payment will be issued on 2023-03-09. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@yuwenmemon] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@yuwenmemon] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@yuwenmemon] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@yuwenmemon] Propose [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@abekkala] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.78-0 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15578


If no regressions arise, payment will be issued on 2023-03-13. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@yuwenmemon] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@yuwenmemon] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@yuwenmemon] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@yuwenmemon] Propose [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@abekkala] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:
